Stylized Home Screen

sotiriosmoustogiannis opened the pull request:
- Added Background and Logo
- Added 5 Need Help Buttons
- Fixed Chat Service sending delayed voice message

Commit:
Stylized Home Screen
- Added Background and Logo
- Added 5 Need Help Buttons
- Fixed Chat Service sending delayed voice message

kerk12 reviewed (changes requested):
Please replace the Palaemon logo with [this image](https://kgiannakis.me/img/palaemon.9154d279.jpg).

Commit:
Stylized Main Screen
- Changed Palaemon Logo
- Changed font weight of text to bold at the 5 Need Help Buttons (Home Screen)
- Created HEX colors as MaterialColor type
- Changed the primary swatch color attr of Theme Data to Palaemon's logo colors
